R/verb-copy-to.R

```diff
@@ -87,6 +87,120 @@ copy_to.src_sql <- function(dest, df, name = deparse(substitute(df)),
   invisible(out)
 }
 
+#' @export
+db_values <- function(con, df) {
```

Comment:
`db_values()` doesn't seem to be a terribly evocative name to me. It feels like the right name should somehow be related to `copy_to()`. Or maybe you should be able to request values from with an argument to `copy_to()`? (This could would stay as is, we'd just keep `db_values()` internal)

Reply:
Imho the following speaks against adding an argument to `copy_to()`:
* Many arguments do not make sense: `name` (would only make sense in a CTE), `overwrite`, `temporary`, `unique_indexes`, `indexes`, `analyze`, `in_transaction`
* The naming is a bit weird to me. On the one hand it allows using the local table in the database but on the other hand it does not really copy it to the database.
* I always find it more difficult to use method specific arguments because it is more difficult to look them up and there is no autocomplete

Maybe something like `copy_to_inline()` or `copy_to_sql()`?

Reply:
Those arguments make sense to me.

Maybe `copy_inline()`? Or `bundle_to()`?